content/post/council-meeting-2022-07-18/chart.csv

```diff
@@ -1,26 +1,25 @@
-http://a2gov.legistar.com/LegislationDetail.aspx?ID=5716033&GUID=C591C535-E66E-45B9-9C10-3B8BACB92506&Options=&Search=,resolution pass,DC-1,Resolution Directing the City Administrator to Expand Compliance Evaluations and to Continue to Provide Annual Reports Regarding Contractor Compliance with Prevailing Wage Requirements,TRUE,TRUE,FALSE,TRUE,TRUE,TRUE,TRUE,absent,FALSE,TRUE,FALSE
-http://a2gov.legistar.com/LegislationDetail.aspx?ID=5716033&GUID=C591C535-E66E-45B9-9C10-3B8BACB92506&Options=&Search=,resolution amendment fail,DC-1 Postponement,Postponing DC-1 until August 4th,FALSE,FALSE,TRUE,FALSE,TRUE,FALSE,FALSE,absent,TRUE,FALSE,TRUE
-http://a2gov.legistar.com/LegislationDetail.aspx?ID=5723495&GUID=84DB57E4-5CF7-41B4-BF91-CBCBD69C9FE9&Options=&Search=,resolution pass,DC-2,"Resolution to Organize a Joint Meeting of Interested Individuals from the Environmental Commission, Transportation Commission, Energy Commission, A2ZERO Ambassadors, and other Community Stakeholders to Identify Opportunities for Greater Coordination around Transportation, Pedestrian Safety, and Sustainability Initiatives Related to Sidewalk Expansion and Maintenance",TRUE,TRUE,TRUE,FALSE,TRUE,TRUE,TRUE,absent,TRUE,TRUE,TRUE
-http://a2gov.legistar.com/LegislationDetail.aspx?ID=5723495&GUID=84DB57E4-5CF7-41B4-BF91-CBCBD69C9FE9&Options=&Search=,resolution amendment pass,DC-2 Amendment,Add commission on disabilities to the list of groups involved in the special meeting,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,absent,TRUE,TRUE,TRUE
-http://a2gov.legistar.com/LegislationDetail.aspx?ID=5723495&GUID=84DB57E4-5CF7-41B4-BF91-CBCBD69C9FE9&Options=&Search=,resolution amendment fail,DC-2 Amendment,Add Hickory Lane (Racket Club) sidewalk project to the list of stuff for the group to work on.,FALSE,FALSE,FALSE,FALSE,TRUE,FALSE,recused,absent,TRUE,FALSE,FALSE
-http://a2gov.legistar.com/LegislationDetail.aspx?ID=5723495&GUID=84DB57E4-5CF7-41B4-BF91-CBCBD69C9FE9&Options=&Search=,resolution amendment pass,DC-2 Recusal,Recusal for CM Grand due to racket club interest in Hickory Lane,TRUE,TRUE,FALSE,TRUE,TRUE,TRUE,TRUE,absent,TRUE,TRUE,FALSE
-http://a2gov.legistar.com/LegislationDetail.aspx?ID=5723495&GUID=84DB57E4-5CF7-41B4-BF91-CBCBD69C9FE9&Options=&Search=,resolution amendment pass,DC-2 Amendment,Remove final resolved clause for the city administrator to evaluate members of commissions.,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,absent,TRUE,TRUE,TRUE
+https://a2gov.legistar.com/LegislationDetail.aspx?ID=5706387&GUID=C52C4AD3-67F8-40B5-8C0B-4804FA72BB37&G=505481FE-B2ED-4B34-9A92-FDB26536D1E1&Options=&Search=,nomination pass,MC-1,Appointment - Confirmations,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE
+https://a2gov.legistar.com/LegislationDetail.aspx?ID=5721531&GUID=22FCE356-261C-4565-A052-EA5A628EB387&G=505481FE-B2ED-4B34-9A92-FDB26536D1E1&Options=&Search=,nomination amendment,MC-2,Appointments - Confirmations,,,,,,,,,,,
```

Comment:
for some reason it thought that this was an amendment

Reply:
Yeah, it assumes any action other than "approved" should get the "amendment" css class. In this case I think the action was "referred", meaning it would get a vote in a subsequent meeting, but not this one. That's generally how nominations/appointments work - they're presented in one meeting, then voted on in the next.

So assuming all of that is true, then MC-2 really shouldn't appear on the chart at all. However, it's difficult to programmatically determine that solely from the (pre-meeting) agenda. Much easier to do it after the meeting is over and votes have been taken, i.e. only show rows which have a non-null "EventItemPassedFlag".